src/App.vue

```diff
@@ -4,18 +4,28 @@ import SidePanel from '@/components/home/SidePanel.vue'
 import { RennesApp } from '@/services/RennesApp'
 import MapComponent from '@/components/map/MapComponent.vue'
 import mapConfig from './map.config.json'
+import { useViewsStore } from './stores/views'
 import UiButtonWithTooltip from '@/components/ui/UiButtonWithTooltip.vue'
 
+import { viewList } from './model/views.model'
+
+const viewStore = useViewsStore()
+
 onBeforeMount(() => {
   const rennesApp = new RennesApp(mapConfig)
   provide('rennesApp', rennesApp)
 })
+
+function isLeftPanelRetractable() {
+  const retractableList = [viewList['map-pcaet']]
+  return retractableList.includes(viewStore.currentView)
```

Comment:
If the left pannel is non retractable only on Home view, maybe replace this function by:
const nonRetractableList = [viewList['home']]
return !nonRetractableList.includes(viewStore.currentView)

When we create a new view it will be by default with a retractable side panel